querybook/webapp/lib/data-doc/datadoc-permission.ts

```diff
@@ -20,33 +22,40 @@ export function readWriteToPermission(
     read: boolean,
     write: boolean,
     isOwner: boolean,
-    publicDoc: boolean
+    publicDoc: boolean,
+    id: number
```

Comment:
instead of id field, it should be a boolean to indicate if its a permissionFromGroup